### PR TITLE
Updated prepobs and fit2obs versions for Orion Rocky9

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
# Description

This PR updates global-workflow to use the new `prepobs/v1.0.2` and `fit2obs/v1.1.2`, which now support Orion Rocky9. Updates are included for new install locations on WCOSS2, Hera, Orion/Hercules, and Jet.

@InnocentSouopgui-NOAA was notified for supporting S4.

The `prepobs_run_ver=v1.0.2` and `fit2obs_ver=v1.1.2` versions are updated, as well as `MODULEPATH`s for new installs that use version variables. Temporary location on Orion/Hercules is set ahead of @WalterKolczynski-NOAA returning and installing new packages under glopara space on Orion.

Refs #2694

# Type of change

External component updates

# How has this been tested?

Low-res atmos-only 5.5 cycle test on WCOSS2-Dogwood, Hera, Orion, Hercules.

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
looks good.

File: modulefiles/module_base.hercules.lua [hunk 1]
```diff
@@ -43,10 +43,12 @@ setenv("WGRIB2","wgrib2")
 setenv("WGRIB","wgrib")
 setenv("UTILROOT",(os.getenv("prod_util_ROOT") or "None"))
 
-prepend_path("MODULEPATH", pathJoin("/work/noaa/global/glopara/git/prepobs/feature-GFSv17_com_reorg_log_update/modulefiles"))
+--prepend_path("MODULEPATH", pathJoin"/work/noaa/global/glopara/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles")
+prepend_path("MODULEPATH", pathJoin("/work/noaa/global/kfriedma/glopara/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles"))
 load(pathJoin("prepobs", (os.getenv("prepobs_run_ver") or "None")))
 
-prepend_path("MODULEPATH", pathJoin("/work/noaa/global/glopara/git/Fit2Obs/v" .. (os.getenv("fit2obs_ver") or "None"), "modulefiles"))
+--prepend_path("MODULEPATH", pathJoin("/work/noaa/global/glopara/git/Fit2Obs/v" .. (os.getenv("fit2obs_ver") or "None"), "modulefiles"))
```
Comment (DavidHuber-NOAA) [inline modulefiles/module_base.hercules.lua:50]: I believe the plan is to move this to glopara when Walter comes back, correct?

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good, just one question.

File: modulefiles/module_base.hercules.lua (hunk 1 above)
Comment (KateFriedman-NOAA) [reply modulefiles/module_base.hercules.lua:50]: Yep! I've left the path we'll move to commented out. We'll move back to the glopara space for prepobs and fit2obs on Orion and Hercules when Walter is back.

**Comment by InnocentSouopgui-NOAA** [comment]
For S4, I will take care of it soon.
There is a plan to update S4 to Rocky 8 this summer. I will bundle this update with the migration to Rocky8.
If a User needs it sooner, I will do the modifications earlier.

**Comment by KateFriedman-NOAA** [comment]
> For S4, I will take care of it soon. There is a plan to update S4 to Rocky 8 this summer. I will bundle this update with the migration to Rocky8. If a User needs it sooner, I will do the modifications earlier.

Sounds like a plan, thanks @InnocentSouopgui-NOAA !

**Comment by aerorahul** [comment]
@KateFriedman-NOAA 
Any idea on the failed Hercules test?

**Comment by DavidHuber-NOAA** [comment]
@aerorahul This is due to bad fix files on Hercules.  Details are in slack.

**Comment by aerorahul** [comment]
Since WCOSS2 and Hera have passed, and Hercules failure cause is known, I suggest we merge.
The hercules failure occurred because the fix files were touched accidentally by a user.
This supports the need a group account on MSU systems

**Pull request merged**